### DISCUSSION TITLE
How to upgrade OS to the latest for Windows and Linux node?

### DISCUSSION THREAD

**Discussion by MANUDEV-H-MENON** [rancher/rke2#6234] [General]
How do we upgrade OS to the latest/specific version for Windows and Linux nodes ?
Is it possible through system-upgrade-controller ?
I had gone though https://github.com/rancher/system-upgrade-controller and couldn't find leads on OS upgrades.
It talks about kernel version upgrade on Ubuntu OS , but not about the  Windows OS upgrade or Linux versions.
Any suggestion, recommendation or reference will be really helpful.
Any other approach instead of system-upgrade-controller also works. 

Thanks

**Comment by manuelbuil** [comment]
The system-upgrade-controller is not supposed to upgrade the OS

**Reply by brandond** [reply]
There are examples in the system upgrade controller repo for upgrading various Linux distros, but adapting those examples to  different distros, or to a completely different OS like Windows, is not something we have the cycles to assist with at this time.
